includes/emails/email-summary/edd-email-summary-template.php

```diff
@@ -0,0 +1,410 @@
+				}
+				.logo-holder {
+					padding: 10px 22px 5px 22px !important;
+				}
+				.logo-holder .edd-logo {
+					max-width: 175px !important;
+					max-height: 28px !important;
+				}
+				.content-holder {
+					padding: 22px 22px 0px 22px !important;
+				}
+				.content-holder.pro-tip-section {
+					padding: 0px 22px 27px 22px !important;
+				}
+				.period-date {
+					font-size: 10px !important;
+					line-height: 13px !important;
+				}
+				.link-style {
+					font-size: 10px !important;
+					line-height: 13px !important;
+				}
+				.link-style.bigger {
+					font-size: 10px !important;
+					line-height: 13px !important;
+				}
+				.stats-totals-wrapper {
+					margin-top: 38px !important;
+				}
+				.stats-totals-wrapper td {
+					font-size: 12px !important;
+					border-collapse: collapse !important;
+				}
+				.stats-total-item {
+					width: 145px;
+					display: inline-table;
+					min-width: 145px;
+				}
+				.stats-total-item-inner {
+					font-size: 12px !important;
+					border-collapse: collapse !important;
+				}
+				.stats-total-item-title {
+					font-size: 12px !important;
+					line-height: 15px !important;
+					margin-bottom: 2.5px !important;
+				}
+				.stats-total-item-value {
+					line-height: 18px !important;
+					font-size: 16px !important;
+				}
+				.stats-total-item-percent {
+					line-height: 8px !important;
+					font-size: 8px !important;
+					white-space: nowrap;
+				}
+				.stats-total-item-percent span.comparison {
+					font-style: normal;
+					font-weight: lighter;
+					font-size: 8px !important;
+					line-height: 10px !important;
+					margin-top: 3px !important;
+				}
+				.stats-total-item-percent img {
+					width: 8px !important;
+					height: 6px !important;
+					vertical-align: baseline !important;
+				}
+				.table-top-title {
+					font-size: 12px !important;
+					line-height: 15px !important;
+				}
+				table.top-products tr th {
+					padding: 6px 0px !important;
+					font-size: 10px !important;
+					line-height: 13px !important;
+				}
+				table.top-products tr td {
+					font-size: 10px !important;
+					padding: 6px 0px !important;
+					border-bottom: 1px solid #F0F1F4;
+				}
+				table.top-products tr td:nth-child(2) {
+					font-size: 12px !important;
+				}
+				.pro-tip-holder {
+					background: #F3F8FE;
+					border-radius: 8px !important;
+					padding: 24px 24px !important;
+				}
+				.pro-tip-section-title {
+					font-size: 14px !important;
+					line-height: 15px !important;
+				}
+				.pro-tip-section-title img {
+					margin-right: 3px !important;
+				}
+				.pro-tip-title {
+					font-size: 12px !important;
+					line-height: 16px !important;
+				}
+				.cta-btn {
+					padding: 6px 15px;
+					font-weight: 600;
+					font-size: 14px;
+					line-height: 20px;
+					background: #2794DA;
+					display: inline-block;
+					color: #FFFFFF;
+					text-decoration: none;
+				}
+			}
+		</style>
+
+		<body style="margin: 0px;">
```

Comment:
@zkawesome We need to set a max width on this, so it doesn't span the whole body area in the email client. Also, once we do that we can set the margins to be `0px auto` so that it is centered in the email client.

In the Figma design for this the max width on desktop is `450px` and on mobile it is`320px`

We can use a media query on this for screen devices with a max width of `480px`